### PR TITLE
[bitnami/rabbitmq] Choice of enabling loopbackUser

### LINKED ISSUE #14445: [bitnami/rabbitmq] Choice of enabling guest-user
### Name and Version

bitnami/rabbitmq 11.4.0

### What is the problem this feature will solve?

There might be a need to enable the guest user. guest user is currently disabled without an option to enable the guest user.
it currently is hardcoded to `false`
```
configuration: |-
  # enable guest user
  loopback_users.guest = false
```

### What is the feature you are proposing to solve the problem?

A new parameter could be used to opt-in the guest-user-access.
Of course the default stays on `false`.

Possible change.
```
auth: 
  ## @param Enable guest user access? Be advised, that enabling the guest user is a possible security risk!
  ##
  enableGuestUser: false



configuration: |-
  # enable guest user
  loopback_users.guest = {{ .Values.auth.enableGuestUser }}

```

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by hgm-kklingspor**
Fixes #14445
Signed-off-by: Karsten Klingspor <karsten.klingspor@georg.com>

### Description of the change

This PR adds the possibility to enable loopbackAccess for the chosen default user
It adds the following parameter:

| Name                                  | Description                                                                                                                                                             | Value                                             |
| ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
| `auth.enableLoopbackUser`             | Enable Loopback user access? Be advised, that enabling the loopback user is a possible security risk!                                                                   | `false`                                           |
### Benefits

Sometimes access to the management-webInterface is needed. This webInterface may only be accessible through localhost. therefore a valid loopbackUser is necessary.

### Possible drawbacks

Activating loopback-access might be a security-risk according to the official rabbitmq-documentation. ref: https://www.rabbitmq.com/configure.html

According to this, access is disabled by default.

### Applicable issues

  - fixes #14445 

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.4.0
+version: 11.4.1
```
Comment (Mauraza) [inline bitnami/rabbitmq/Chart.yaml:26]: As it is something new, I think it should be a minor.
```suggestion
version: 11.5.0
```

**Review by Mauraza — COMMENTED** [review]
Hi @hgm-kklingspor, 

Thanks for the contribution! Could you check my comment?

File: bitnami/rabbitmq/Chart.yaml (hunk 1 above)
Comment (hgm-kklingspor) [reply bitnami/rabbitmq/Chart.yaml:26]: Seems reasonable to me.

Do I have to make a new commit regarding this change?
Sorry, this is my first time contributing.

Comment (Mauraza) [reply bitnami/rabbitmq/Chart.yaml:26]: You can click the button "Sign off and commit suggestion" to resolve the comment, after that I can merge the PR

**Review by Mauraza — APPROVED** [review]
Hi @hgm-kklingspor!!

Thanks again for the contribution. LGTM!

**Pull request merged**